src/myspace/PersonalDetails.jsx

```diff
@@ -6,8 +6,12 @@ import "./MySpace.css";
 
 export default function PersonalDetails() {
   const [errorMessage, setErrorMessage] = useState(null);
+  const [responseMessage, setResponseMessage] = useState(null);
   const [personalDetails, setPersonalDetails] = useState({});
   const { access_token } = useContext(AuthContext);
+  const [validName, setValidName] = useState(true);
```

Comment:
use isValidName to name booleans. 
booleans are named as isSomething, hasSomething

Reply:
Fone. Refactored to IsValidName and isValidUsername